scripts/initialize-celo.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+set -euo pipefail
+
+KEEP_CORE_PATH_DEFAULT=$(realpath -m $(dirname $0)/../../keep-core)
+CONFIG_DIR_PATH_DEFAULT=$(realpath -m $(dirname $0)/../configs)
+KEEP_ECDSA_PATH=$(realpath $(dirname $0)/../)
+KEEP_ECDSA_SOL_PATH=$(realpath $KEEP_ECDSA_PATH/solidity)
+
+# Defaults, can be overwritten by env variables/input parameters
+KEEP_CELO_PASSWORD=${KEEP_CELO_PASSWORD:-"password"}
+NETWORK_DEFAULT="local"
+CONTRACT_OWNER_CELO_ACCOUNT_PRIVATE_KEY=${CONTRACT_OWNER_CELO_ACCOUNT_PRIVATE_KEY:-""}
+
+help()
+{
+   echo -e "\nUsage: ENV_VAR(S) $0"\
+           "--keep-ecdsa-config-path <path>"\
+           "--application-address <address>"\
+           "--network <network>"
+   echo -e "\nEnvironment variables:\n"
+   echo -e "\tCONTRACT_OWNER_CELO_ACCOUNT_PRIVATE_KEY: Contracts owner private key on Celo"
+   echo -e "\nCommand line arguments:\n"
+   echo -e "\t--keep-ecdsa-config-path : Path to keep-ecdsa client configuration file(s)"
+   echo -e "\t--application-address: Address of application approved by the operator"
+   echo -e "\t--network: Celo network for keep-ecdsa client."\
+           "Available networks and settings are specified in 'truffle.js'\n"
+   exit 1 # Exit script after printing help
+}
+
+# Transform long options to short ones
+for arg in "$@"; do
+  shift
+  case "$arg" in
+    "--keep-ecdsa-config-path")    set -- "$@" "-d" ;;
+    "--application-address")       set -- "$@" "-a" ;;
+    "--network")                   set -- "$@" "-n" ;;
+    "--help")                      set -- "$@" "-h" ;;
+    *)                             set -- "$@" "$arg"
+  esac
+done
+
+# Parse short options
+OPTIND=1
+while getopts "d:a:n:h" opt
+do
+   case "$opt" in
+      d ) config_dir_path="$OPTARG" ;;
+      a ) client_app_address="$OPTARG" ;;
+      n ) network="$OPTARG" ;;
+      h ) help ;;
+      ? ) help ;; # Print help in case parameter is non-existent
+   esac
+done
+shift $(expr $OPTIND - 1) # remove options from positional parameters
+
+CONFIG_DIR_PATH=${config_dir_path:-$CONFIG_DIR_PATH_DEFAULT}
+KEEP_ECDSA_CONFIG_DIR_PATH=$(realpath $CONFIG_DIR_PATH)
+NETWORK=${network:-$NETWORK_DEFAULT}
+
+cd $KEEP_ECDSA_SOL_PATH
+
+if [ "$NETWORK" == "local" ]; then
+  # Dafault app address.
+  output=$(npx truffle exec scripts/get-default-application-account.js --network $NETWORK)
+  CLIENT_APP_ADDRESS=$(echo "$output" | tail -1)
+fi
+
+if [ ! -z ${client_app_address+x} ]; then
+  # Read user app when --application-address is set
+  CLIENT_APP_ADDRESS=$client_app_address
+fi
+
+# Read application address in case it was not set from the command line
+if [ -z ${CLIENT_APP_ADDRESS+x} ]; then
+  read -p "Enter client application address: " client_app_address
+  CLIENT_APP_ADDRESS=${client_app_address}
+fi
```

Comment:
Is this possible to happen? We are setting it in `get-default-application-account.js`.

Reply:
Yes, if you're on `Alfajores` network, then `get-default-application-account.js` won't be executed. If you won't provide an application address in a command line, then it will ask you to enter it here. But if we are leaning towards handling only `local` network as part of initialisation, then this `if` block won't be necessary.

Reply:
Ah! I was looking at `initialize.sh` and it behaves differently from `initialize-celo.sh` when it comes to `get-default-application-account.js`. If the changes we discussed allows keeping the behavior aligned, that's only better.

Reply:
This block can now be removed, correct?

Reply:
b553d44 yes, we can. In this case an application account on Alfajores will be either accounts[0] or an address passed in `--application-address`